sdk/python/kfp/v2/compiler/compiler_test.py

```diff
@@ -70,6 +71,134 @@ def simple_pipeline(pipeline_input='Hello KFP!',):
     finally:
       shutil.rmtree(tmpdir)
 
+  def test_compile_pipeline_with_dsl_condition_should_raise_error(self):
+
+    class FlipCoinOp(dsl.ContainerOp):
```

Comment:
Can you please replace this with components? Inheriting from `ContainerOp` or calling `ContainerOp` constructor is deprecated.
Adding more cases increases the technical debt.

Reply:
Done